Bucketeer/Sources/Internal/Scheduler/EvaluationForegroundTask.swift

```diff
@@ -20,11 +20,10 @@ final class EvaluationForegroundTask: ScheduledTask {
         self.maxRetryCount = maxRetryCount
     }
 
-    private func reschedule(isRetrying: Bool) {
+    private func reschedule(interval: Int64) {
```

Comment:
Refactored this function to keep the consistency with Android SDK.
Reference: https://github.com/bucketeer-io/android-client-sdk/blob/main/bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/scheduler/EvaluationForegroundTask.kt#L23-L31